internal/service/loadbalancer/backend.go

```diff
@@ -306,5 +333,37 @@ func schemaBackendProperties() map[string]*schema.Schema {
 				}, false),
 			),
 		},
+		"tls_configs": {
+			Description: "Set of TLS config names",
+			Type:        schema.TypeList,
+			Computed:    true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
+		"tls_enabled": {
+			Description: "Enables TLS connection from the load balancer to backend servers.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+		},
+		"tls_verify": {
+			Description: "Enables backend servers certificate verification. Please make sure that TLS config with the certificate bundle of type authority attached to the backend or tls_use_system_ca enabled. Note: tls_verify has preference over health_check_tls_verify when tls_enabled in true.",
```

Comment:
Should we use backticks to highlight the related properties, e.g. `tls_use_system_ca`, in the docs. Not sure if we have done that earlier or if that is supported by the docs system, though 🤔